src/flightlog.js

```
@@ -869,6 +875,39 @@ export function FlightLog(logData) {
     return fieldIndex;
   };
 
+  /**
+   * Compute sum of plane SAS channels components.
+   * Writes 3 field to destFrame at fieldIndex.
+   * Returns updated fieldIndex.
+   */
+  const computePlaneSasSum = (srcFrame, destFrame, fieldIndex, psasData) => {
+    let psasRollSum = 0;
+    for (const rollDataIndex of psasData[AXIS.ROLL]) {
+      if (rollDataIndex != undefined) {
```

ESLint

[error] Expected '!==' and instead saw '!='.

(eqeqeq)